includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -183,6 +184,20 @@ public function sanitize() {
 				continue;
 			}
 
+			// Replace img with amp-pixel when dealing with tracking pixels.
+			if ( self::is_tracking_pixel_url( $node->getAttribute( Attribute::SRC ) ) ) {
+				$amp_pixel_node = AMP_DOM_Utils::create_node(
+					$this->dom,
+					'amp-pixel',
```

Comment:
You could add `use AmpProject\Extension;` to the top and then here do:

```suggestion
					Extension::PIXEL,
```

Reply:
Good idea. Plus, we already have `use AmpProject\Extension;` at the top. This has been addressed in 0d66f37.